### ISSUE TITLE
Website - Main Steps Page – Poor Color Contrast of Step Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#64]
When people hover over the individual Steps the text that show is gray text on light gray background. The color contrast ratio for the gray text on the light gray background is 1.98:1 and needs to be 4.5:1 or higher to pass.

See attached screenshot.

![complychain - main steps - poor color contrast on hover](https://user-images.githubusercontent.com/10423487/30119671-86b1156a-92f4-11e7-8f47-0fcbb2e669dc.png)

**Comment by adriennerlong** [comment]
ICF, can you make the text dark grey/black? Please let me know if you need our graphic designer to re-do the individual steps with the different colored text,

**Comment by jfc3-dol** [comment]
The issue does not persist over all three platforms (iOS, Android, and web). It is only an issue for web.

**Comment by JckICF** [comment]
@adriennerlong If you could have your graphic designer re-do the indvidual steps that would be great.

**Comment by adriennerlong** [comment]
@JckICF Thanks for letting us know. We won't be able to have the graphic designer get to this by the 9/20 launch, but she will work on it for a future update.

**Comment by adriennerlong** [comment]
From ILAB: Our contract with our graphic designer has ended so this is something we are not prepared to address right now. However, we can make this a priority in the next re-vamp of the app (expected 2018) once we have a new contract with our graphic designers.

**Comment by jfc3-dol** [comment]
@adelevie,

Okay, will leave this open so we have it as a reference for the next version of ComplyChain.

**Comment by jfc3-dol** [comment]
The new Main Steps image and messages of what step a person is on now have enough color contrast for the web, iOS, and Android versions of the application.

Closing issue.
